### PR TITLE
Develop js_task_clone_table_column_DOM

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
[DEMO LINK](https://sergey-lipikhin.github.io/js_task_clone_table_column_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+function CloneElementOfRow(row) {
```
Comment (sergii-nosachenko) [inline src/scripts/main.js:3]:
```suggestion
function CloneElementOfRows(rows, from, to) {
```
Pass an array here and loop through it.

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+const tableHeadElement = tableElement.tHead.rows[0];
+const tableBodyRowElements = tableElement.tBodies[0].rows;
+const tableFootElement = tableElement.tFoot.rows[0];
+
+const columnIndexToClone = 1;
+const columnIndexRecipient = tableHeadElement.cells.length - 1;
+
+CloneElementOfRow(tableHeadElement);
+CloneElementOfRow(tableFootElement);
+[...tableBodyRowElements].forEach(CloneElementOfRow);
```
Comment (sergii-nosachenko) [inline src/scripts/main.js:21]:
```suggestion
[tHead, ...tBodies, tFoot].forEach(
  ({ rows }) => CloneElementOfRows(rows, columnIndexToClone, columnIndexRecipient),
);
```

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]
Can be improved

**Review by illiarerun — APPROVED** [review]
GJ

**Pull request closed without merging**